docs/installation.rst

```diff
@@ -58,6 +59,7 @@ Raspbian     `Raspbian testing`_                           ``apt install borgbac
 Ubuntu       `Ubuntu packages`_, `Ubuntu PPA`_             ``apt install borgbackup``
 ============ ============================================= =======
 
+.. _Alpine repository: https://pkgs.alpinelinux.org/packages?name=borgbackup&branch=edge
```

Comment:
what's the meaning of branch edge?

Reply:
We don't need it, can just reference:

https://pkgs.alpinelinux.org/packages?name=borgbackup